nss-day-cohort-60/rare-client-tenacious-tators
# Evn buttons

# Description of Proposed Changes
Styled buttons for continuity on Subscription List
Took white space out of Navbar links in normal and hamburger menu
---


# Steps to Test

1. Fetch the `EVN-buttons` branch and switch to it
2. Check that background color remains dark when hovering over navbar links, but text is light
3. Check buttons on homepage for comments are styled rounded and blue like all others
